cmd/elf2c/elf2c.go

```diff
@@ -136,6 +158,17 @@ func main() {
 		fmt.Fprintf(os.Stderr, "gencode\n")
 		_, file := path.Split(n)
 		fmt.Fprintf(w, "uintptr_t %v_main = %v;\n", n, f.Entry)
+		var msg string
+		if codestart == -1 {
+			msg = fmt.Sprintf("codestart was never set for %s; the ELF file has no 'R E' segment.", file)
+		}
+		if datastart == -1 {
+			msg = fmt.Sprintf("datastart was never set for %s; the ELF file has no 'RW' segment", file)
+		}
+		if len(msg) > 0 {
+			log.Fatalf(msg)
```

Comment:
`Fatalf` expects a format string. `Fatal` would be safer.
```suggestion
			log.Fatal(msg)
```

Reply:
done